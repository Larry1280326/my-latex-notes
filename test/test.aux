\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}2D plots}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}3D plots}{2}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}vectors}{3}{section.3}\protected@file@percent }
\gdef \@abspage@last{3}
