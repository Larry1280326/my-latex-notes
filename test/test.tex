\documentclass[11pt]{article}
\usepackage[]{amsfonts, amssymb, amsmath, float, hyperref,fancyheadings, graphicx, derivative}
\usepackage[utf8]{inputenc}
\usepackage{pgfplots}
\pgfplotsset{compat=1.18, width=8cm}
\usepackage{geometry}
\geometry{
a4paper,
total={170mm,257mm},
left=20mm,
top=20mm,
}
\title{Making plots in \latex}
\author{Larry}
\date{Aug 2024}
\begin{document}

\tableofcontents
\section{2D plots}
\begin{tabular}{cc}
\begin{tikzpicture}
\begin{axis}[xmin=-2, xmax=2, ymin=-2, ymax=2, 
axis lines= middle, 
xlabel= $x$, 
ylabel= $y$,
title = {2D plot Sample 1}]

\addplot[color = red, dashed, samples = 100, domain = -1: 1]{x^2};
\addplot[color = blue, dotted, samples = 100]{1- x^2};

\end{axis}
\end{tikzpicture}&
\begin{tikzpicture}

\begin{axis}[clip = false,
axis lines= middle,
xmin = 0,
xmax = 2.5*pi,
ymin= -1.5,
ymax = 1.5,
title = {2D plot Sample 2},
xtick = {0, pi/2, pi, 1.5*pi, 2*pi},
xticklabels={$0$, $\frac{\pi}{2}$,$\pi$, $\frac{3\pi}{2}$, $2\pi$},
xticklabel style = {anchor= south west},
xmajorgrids = true,
grid style = dashed]

\addplot[domain= 0:2*pi, color = red]{sin(deg(x))}{node[right, pos=0.3]{$f(x) = \sin x$}};
\addplot[domain= 0:2*pi, color = blue]{cos(deg(x))}{node[right, pos=0.5]{$f(x) = \cos x$}};

\end{axis}
\end{tikzpicture}\\

\begin{tikzpicture}
\begin{axis}[
title= {Scatter plot of GPA versus Ages},
xlabel = $GPA$, 
ylabel = $Ages$]
\addplot+[
only marks,
scatter,
mark size= 1.5 pt]
table[meta= Ages]{test.txt};
\end{axis}
\end{tikzpicture}&

\begin{tikzpicture}
\begin{axis}[
title= {Scatter plot of GPA versus Wages},
xlabel = $GPA$, 
ylabel = $Wages$]
\addplot+[
only marks,
scatter,
mark size= 1.5 pt]
table[meta= Wages]{test.txt};
\end{axis}
\end{tikzpicture}\\

\begin{tikzpicture}

\begin{axis}[ybar stacked, title = {ybar stacked plot}]
\addplot coordinates
{(0, 1) (1, 2) (2, 3)};
\addplot coordinates
{(0, 3) (1, 3) (2, 8)};
\addplot coordinates
{(0, 4) (1, 7) (2, 9)};
\end{axis}

\end{tikzpicture}

\end{tabular}

\section{3D plots}
\begin{tabular}{cc}

\begin{tikzpicture}
\begin{axis}[title= {default},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$]
\addplot3 [] {1- x^2 - y^2};
\end{axis}
\end{tikzpicture}&

\begin{tikzpicture}
\begin{axis}[title = {mesh},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$]
\addplot3 [mesh] {1- x^2 - y^2};
\end{axis}
\end{tikzpicture}\\

\begin{tikzpicture}
\begin{axis}[title= {surf},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$]
\addplot3 [surf] {1- x^2 - y^2};
\end{axis}
\end{tikzpicture}&

\begin{tikzpicture}
\begin{axis}[title= {vector-valued function},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$]
\addplot3+[domain= 0:5*pi, samples= 60, samples y= 0]
({sin(deg(x))},
{cos(deg(x))},
{x});
\end{axis}
\end{tikzpicture}\\

\begin{tikzpicture}
\begin{axis} [title = {$z = x^2 + y^2$},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$
]
\addplot3 [mesh]{x^2 + y^2};
\end{axis}
\end{tikzpicture}&

\begin{tikzpicture}
\begin{axis} [title = {$z = \sin x + \cos y$},
xlabel = $x$,
ylabel = $y$,
zlabel = $z$
]
\addplot3 [mesh]{sin(deg(x)) + cos(deg(y))};
\end{axis}
\end{tikzpicture}
\end{tabular}

\section{vectors}
\begin{tabular}{cc}
\begin{tikzpicture}[scale = 1]
\draw[thin,gray!20] (-5,-4) grid (5, 4);
\draw (5, 0) -- (-5, 0) (0, 4) -- (0, -4);
\draw [->, ultra thick, color = red] (0, 0) -- (3,3) {node[left, pos=0.9]{$\vec{v} = <3, 3>$}};
\draw [->, ultra thick, color = blue] (0, 0) -- (-4, -2) {node[right, pos=0.9]{$\vec{w} = <-4, -2>$}};
\draw [->, ultra thick, color = green] (0, 0) -- (-1, 1) {node[left, pos=0.9]{$\vec{v} + \vec{w} = <-1, 1>$}};
\draw [->, ultra thick, color = blue, dashed] (3, 3) -- (-1, 1);
\draw [->, ultra thick, color = red, dashed] (-4, -2) -- (-1, 1);
\end{tikzpicture}&
\begin{tikzpicture}
\draw [-latex] (0, 0, 0) -- (5, 0, 0) node[left]{$x$};
\draw [-latex] (0, 0, 0) -- (0, 5, 0) node[left]{$y$};
\draw [-latex] (0, 0, 0) -- (0, 0, 5) node[left]{$z$};
\draw [->, ultra thick, blue] (0, 0, 0) -- (3, 2, 1) node[pos=0.7, right] {$\vec{a} = <3, 2, 1>$};
\draw [dashed, - , blue] (3, 0, 0) -- (3, 0, 1);
\draw [dashed, - , blue] (0, 0, 1) -- (3, 0, 1);
\draw [dashed, - , blue] (0, 2, 0) -- (3, 2, 1);
\draw [dashed, - , blue] (3, 0, 1) -- (3, 2, 1);
\end{tikzpicture}
\end{tabular}




\end{document}