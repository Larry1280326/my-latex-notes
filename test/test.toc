\contentsline {section}{\numberline {1}2D plots}{1}{section.1}%
\contentsline {section}{\numberline {2}3D plots}{2}{section.2}%
\contentsline {section}{\numberline {3}vectors}{3}{section.3}%
