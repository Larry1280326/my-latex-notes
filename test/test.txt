GPA	Ages	Wages	Number_of_Siblings
2.75	24	62107	0
3.9	18	40666	3
3.46	21	64357	2
3.2	21	71663	2
2.31	22	35708	4
2.31	24	46370	2
2.12	24	69811	2
3.73	28	22811	2
3.2	21	76250	1
3.42	24	67011	4
2.04	28	26546	0
3.94	20	78713	3
3.66	23	54754	0
2.42	29	50206	4
2.36	19	61106	3
2.37	27	31411	4
2.61	26	22911	2
3.05	22	21734	3
2.86	23	38227	2
2.58	29	28680	0
3.22	29	39360	0
2.28	29	38343	3
2.58	29	25759	3
2.73	21	43078	4
2.91	28	55153	4
3.57	27	31111	2
2.4	24	57504	3
3.03	29	21802	0
3.18	26	28155	4
2.09	24	28120	4
3.22	18	59384	0
2.34	18	67025	4
2.13	26	67254	2
3.9	28	41918	3
3.93	26	40445	0
3.62	21	50306	3
2.61	26	36646	4
2.2	20	66843	4
3.37	24	75439	0
2.88	23	51065	2
2.24	25	45199	1
2.99	28	61976	0
2.07	26	36371	1
3.82	22	31835	1
2.52	18	65936	2
3.33	20	22049	1
2.62	27	51616	1
3.04	29	58191	2
3.09	25	67357	1
2.37	28	40932	1
3.94	23	72260	1
3.55	25	49855	0
3.88	26	27158	0
3.79	21	63016	0
3.2	18	27400	2
3.84	18	72322	4
2.18	27	62642	1
2.39	21	35151	1
2.09	29	71407	2
2.65	24	21154	1
2.78	19	24499	0
2.54	20	26295	4
3.66	18	75349	3
2.71	22	79040	1
2.56	18	32183	0
3.09	25	49299	3
2.28	18	32874	4
3.6	28	70800	3
2.15	18	52711	0
3.97	19	25539	3
3.54	19	62405	2
2.4	29	73351	3
2.01	23	77597	1
3.63	24	68354	1
3.41	22	22557	2
3.46	18	58360	0
3.54	18	36482	1
2.15	20	22200	4
2.72	19	66940	1
2.23	22	22961	1
3.73	29	66975	0
3.25	27	41357	3
2.66	23	73585	1
2.13	24	31969	2
2.62	21	22869	3
2.65	24	70108	4
3.46	28	66760	0
3.28	25	48643	4
3.77	28	44736	3
2.94	18	58467	3
2.24	23	43328	3
3.43	25	73645	4
3.52	22	45876	3
3.12	21	41295	4
3.54	19	23987	3
2.99	23	78871	2
3.05	23	42399	3
2.86	28	66214	4
2.05	18	53986	1
2.22	26	40880	3
