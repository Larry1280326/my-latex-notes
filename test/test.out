\BOOKMARK [1][-]{section.1}{\376\377\0002\000D\000\040\000p\000l\000o\000t\000s}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\0003\000D\000\040\000p\000l\000o\000t\000s}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000v\000e\000c\000t\000o\000r\000s}{}% 3
